operator/k8s_identity.go

```diff
@@ -64,12 +65,21 @@ func deleteIdentity(ctx context.Context, identity *v2.CiliumIdentity) error {
 
 var identityHeartbeat *identity.IdentityHeartbeatStore
 
+// counters for GC failed/successful runs
+var (
+	failedRuns     = 0
+	successfulRuns = 0
+)
+
 // identityGCIteration is a single iteration of a garbage collection. It will
 // delete identities that have not had its heartbeat lifesign updated since
 // option.Config.IdentityHeartbeatTimeout
 func identityGCIteration(ctx context.Context) {
 	log.Debug("Running CRD identity garbage collector")
 
+	totalEntries := len(identityStore.List())
+	deletedEntries := 0
+
```

Comment:
This should be done at least after line 84, maybe after line 92.

Reply:
Do you mean inside the for in [#L84](https://github.com/cilium/cilium/blob/master/operator/k8s_identity.go#L84)?
Just to be sure ... every time identityGCIteration() is called I have to count how many identities was deleted at this gargabe collection run, right?